build-system/tasks/extension-helpers.js

```diff
@@ -447,15 +457,20 @@ async function buildExtension(
   if (hasCss) {
     mkdirSync('build');
     mkdirSync('build/css');
-    await buildExtensionCss(path, name, version, options);
+    await buildExtensionCss(extensionPath, name, version, options);
     if (options.compileOnlyCss) {
       return;
     }
   }
+
+  await compileJison(path.join(extensionPath, '**', '*.jison'));
+  if (name === 'amp-bind') {
+    await doBuildJs(jsBundles, 'ww.max.js', options);
+  }
```

Comment:
I like this more because the logs will clearly show that these two are being built together. A case where _post hoc ergo propter hoc_ is actually true!

Reply:
Thank you for pushing for it! It is also no longer part of "runtime files", which IMO is a list we need to trim as well.

I'm surprised at how much cleaner it this ended up being 😄